### ISSUE TITLE
RSPEC-5732 Disabling content security policy frame-ancestors directive is security-sensitive 

### ISSUE THREAD

**Issue by eric-therond-sonarsource** [SonarSource/SonarJS#1952]
Implements [RSPEC-5732](https://jira.sonarsource.com/browse/RSPEC-5732) MMF-1898

For both JS/TS

Should raise on the `csp()`/`helmet()` call (depending on the `helmet-csp`/`helmet` module used) if the `frameAncestors` directive (alias `"frame-ancestors"`) is not set or on the `frameAncestors` directive when set to `["none"]`:

```
var app = express()

  app.use(csp({  // Sensitive: frame-ancestors is not set
    directives: {
      defaultSrc: ["'self'", 'example.com', 'code.jquery.com'],
    }
  }));

  app.use(csp({
    directives: {
      defaultSrc: ["'self'", 'example.com', 'code.jquery.com'],
      "frame-ancestors": ["'none'"] // Sensitive: frame-ancestors is set to none
    } 
  }));
```

when `contentSecurityPolicy` middleware is disabled/set to `false` NO NEED TO RAISE, because [rule RSPEC-5728](https://github.com/SonarSource/SonarJS/issues/1949) will be already triggered:
```
app.use(
    helmet({
      contentSecurityPolicy: false, // Sensitive for 5728 only
    })
  );
```

Code examples: https://github.com/SonarSource/security-expected-issues/tree/master/javascript/express.js/RSPEC-5732

**Comment by eric-therond-sonarsource** [comment]
Reopened because:
* the message is not formatted (see [this issue](https://peach.sonarsource.com/security_hotspots?id=security-expected-issues-express-js_nodev8&hotspots=AXWxbikPMh1ExG5XrVcd) for instance):
![image](https://user-images.githubusercontent.com/56015232/98655382-ac909380-233f-11eb-8369-59595ef7ab2d.png)
* the rule is triggered only one 1 use case [out of 4 identified in security-expected-issues](https://github.com/SonarSource/security-expected-issues/blob/master/javascript/express.js/RSPEC-5732/sensitive.js), we have 3 FNs:
```
  app.use(
    csp({
      directives: {
        defaultSrc: ["'self'", 'example.com', 'code.jquery.com'],
      }, // TP: frame-ancestors is not set
    }),
  );

  // other case sensitive

  app.use(
    csp({
      directives: {
        defaultSrc: ["'self'", 'example.com', 'code.jquery.com'],
        'frame-ancestors': ["'none'"],
      }, // FN: frame-ancestors is set to none
    }),
  );

  // other case sensitive

  app.use(
    helmet.contentSecurityPolicy({
      directives: {
        defaultSrc: ["'self'"],
      }, // FN: "frame-ancestors"/"frameAncestors is not set
    }),
  );

  // other case sensitive

  app.use(
    helmet.contentSecurityPolicy({
      directives: {
        defaultSrc: ["'self'"],
        frameAncestors: ["'none'"],
      }, // FN: "frame-ancestors"/"frameAncestors  is set to none
    }),
  );
```
